app/resources/views/pages/location_search/search.blade.php

```diff
@@ -2,9 +2,11 @@
 @section('head-content')
   @if ( $view === 'map' && !$turn_off_maps  )
   <script>
-    var locations = {!! json_encode($locations) !!};
+    var locations      = {!! json_encode($locations) !!};
 	var user_longitude = {{ $base_user->getLongitude() }};
-	var user_latitude = {{ $base_user->getLatitude() }};
+	var user_latitude  = {{ $base_user->getLatitude() }};
+	var search_radius  = parseInt({{ $search_radius }});
```

Comment:
Use parseFloat instead of parseInt because the search radius can be something like 1.6 and want to make use of those decimals.